fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -647,6 +647,24 @@ public void testGetChildrenWithBinary() {
     }
 
     @Test
+    public void testGetChildrenWithBinaryCustomProperties() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Container container = containerService.findOrCreate(session, "/" + pid);
+        final FedoraResource resource = binaryService.findOrCreate(session, "/" + pid + "/a");
+        final javax.jcr.Node node = resource.getNode();
+        node.setProperty("dc:title", "this-is-some-title");
+
+        final Graph graph =
+                container.getChildren().next().getTriples(subjects, PropertiesRdfContext.class).asModel().getGraph();
```

Comment:
Running this new test in `FedoraResourceImplIT` against the master branch successfully passes... therefore it does not demonstrate a fix to the issue of:
https://jira.duraspace.org/browse/FCREPO-1290

I suggest removing the updates to this class, and instead adding the integration test provided here:
https://github.com/yinlinchen/fcrepo4/pull/4